core/src/components/input/test/item/index.html

```diff
@@ -69,5 +69,15 @@ <h2>Inset List</h2>
         </div>
       </ion-content>
     </ion-app>
+
+    <script>
+      const items = document.querySelectorAll('ion-item');
+
+      for (var i = 0; i < items.length; i++) {
+        items[i].addEventListener('click', function (ev) {
+          console.log('item clicked', ev.target);
```

Comment:
This looks like it was intended for debugging, I'm not sure if you meant to keep it here so you could reference it in the future, I just thought I'd point it out in case you intended to remove it. Either way is fine, since this is just a test file.

Reply:
I mostly left it in to make it easier for testers to see how many click events were fired, but I can remove it if you want.

Reply:
It doesn't matter to me either way. Just with the automated tests it seems unlikely this console log will ever be checked again, so doesn't feel useful, but if you think it could be useful again in the future then it's fine 🤷‍♂️

Reply:
Nah I can remove them, not a big deal.